docs/readthedocs/source/doc/Nano/Howto/accelerate_pytorch_inference_onnx.md

```diff
@@ -0,0 +1,49 @@
+# Accelerate PyTorch Inference using ONNXRuntime
+
+To accelerate your PyTorch inference pipeline through ONNXRuntime as backend engine, BigDL-Nano provides you with `Trainer.trace(accelerator='onnxruntime')` API. Based on that, the optimization could be implemented by applying very few lines of code changes.
```

Comment:
You can use `Trainer.trace(accelerator='onnxruntime')` API to enable the ONNXRuntime acceleration for pytorch inference.  It only takes a few lines.

Reply:
Have revised